src/core/exporter.js

```diff
@@ -92,4 +93,17 @@ function cleanup(cloneDoc) {
   pub("beforesave", documentElement);
 }
 
+/** @param {Document} cloneDoc */
+function prettify(cloneDoc) {
+  cloneDoc.querySelectorAll("style").forEach(el => {
+    el.innerHTML = `\n${el.innerHTML}\n`;
+  });
+  cloneDoc.querySelectorAll("head > *").forEach(el => {
+    el.outerHTML = `\n${el.outerHTML}`;
+  });
+  cloneDoc.querySelectorAll(":root > *").forEach(el => {
+    el.outerHTML = `\n${el.outerHTML}\n`;
+  });
+}
+
```

Comment:
This might be delegated to `tidy` instead and make it opt-in, or keep it pretty by default. WDYT?

Reply:
Pretty by default is good IMO.